x-pack/plugins/triggers_actions_ui/public/application/sections/alert_summary_widget/alert_summary_widget.tsx

```diff
@@ -39,7 +40,10 @@ export const AlertSummaryWidget = ({
     if (!isLoading && onLoaded) {
       onLoaded();
     }
-  }, [isLoading, onLoaded]);
+    if (!isLoading && getAlertsCount) {
+      getAlertsCount({ activeAlertCount, recoveredAlertCount });
```

Comment:
I wonder if we could use the `onLoaded` for this. 🤔

Reply:
I had 2 concerns here: 1. to not break any existing functionality and to avoid confusion with the naming (so you won't expect usually to pass parameters to `onLoaded` rather than execute an action after it is loaded. 
Now that I think about it, it should work - it can be an optional parameter so the current usage won't be changed.